### DISCUSSION TITLE
The detailed documentation for DeepTrack2.1 is missing

### DISCUSSION THREAD

**Discussion by shuyiloe** [DeepTrackAI/DeepTrack2#284] [Resolved Issue]
![image](https://github.com/user-attachments/assets/a624099e-a5d4-46aa-8267-374c543e48c9)

**Comment by BenjaminMidtvedt** [comment]
@shuyiloe 
We're aware! It should be back up soon.

**Comment by giovannivolpe** [comment]
@shuyiloe it's now back up.

Note that we are in the process of uniforming and updating the documentation so it'll improve steadily over the next few weeks.
